src/js/components/CodecBuildSection.jsx

```diff
@@ -655,8 +658,8 @@ const CodecBuildSection = ( {setLogs, hardwareType, setHardwareType} ) => {
                     )
                     : (
                         <>
-                            <Button onClick={onSaveEditedCommandClick} disabled={commandParametersError}>Save</Button>
-                            <Button variant="outlined" onClick={onCancelEditingCommandClick}>Cancel</Button>
+                            <Button id={'saveButton'} onClick={onSaveEditedCommandClick} disabled={commandParametersError}>Save</Button>
+                            <Button id={'cancelButton'} variant="outlined" onClick={onCancelEditingCommandClick}>Cancel</Button>
```

Comment:
```suggestion
                            <Button id="cancel-editing-command-button" variant="outlined" onClick={onCancelEditingCommandClick}>Cancel</Button>
```

and in other similar places

Reply:
switch to data-testid